packages/battery_plus_macos/example/lib/main.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:battery_plus_macos/battery_plus_macos.dart';
+
+void main() {
```

Comment:
I think the example for the macos plugin should be stored in the battery_plus (main plugin) directory (similarly as is in case of connectivity)

Reply:
Yes correct, this example will be removed.